### ISSUE TITLE
Error on trying to analyze sentences with the word 'constructor'

### ISSUE THREAD

**Issue by JustenRickert** [Ulflander/compendium-js#12]
```bash
> compendium = require('compendium-js')
> compendium.analyse('constructor')
TypeError: a.indexOf is not a function
    at Object.token (/Users/justen/myproject/node_modules/compendium-js/dist/compendium.js:3:15453)
    at Object.r.applyPOS (/Users/justen/myproject/node_modules/compendium-js/dist/compendium.js:1:9046)
    at Object.r.analyse (/Users/justen/myproject/node_modules/compendium-js/dist/compendium.js:1:9269)
    at Object.t.analyse (/Users/justen/myproject/node_modules/compendium-js/dist/compendium.js:1:9778)
```

I'm guessing you can't ask for `lexicon.constructor` because that's the constructor method.

**Comment by Ulflander** [comment]
Hi @JustenRickert, thank you for the bug report! The issue has been fixed.

By the way, may I ask how/what you're using Compendium-js for? I'm in the process of working on a new version in Typescript and I'd be glad to have some insights related to how Compendium is used in the wild.

Cheers!

**Comment by JustenRickert** [comment]
No problem! And that's good news about TypeScript, too. It would definitely makes this library easier for me to use if it had a declaration file!

I didn't get too deep into your code here yet, so I haven't come up with something exactly to do with it. Previously I've created (parts of) natural language parsers of pos-tagged sentences, just to study constituency-/dependency-based parse tree and other miscellaneous constructions techniques. Working toward some kind of logical procedural generation.

Your library I've only got around to testing it on my (org-mode) journal entries.
